sources/Outputs/include/Diagram.h

```diff
@@ -76,8 +77,8 @@ class Diagram {
    * @param buffer The buffer to store the string that will be written to the file
    * @param table The enum determining if we write the Qmin or Qmax table
    */
+  static void writeTable(const algo::GeneratorDefinition& generator, std::stringstream& buffer, Tables table);

```

Comment:
why static?

Reply:
It does not depend on any class members, so I figured it would be more optimal